### PR TITLE
[FMS][Open311] Send images when comment on private report

### PR DESCRIPTION
**Pull request by MorayMySoc**
If an image is on a comment which was made on a private report Open311 can not download the image as it will not be visible.

In this case, for Alloy, send the photos directly to open311

Open311 end still to do

https://github.com/mysociety/societyworks/issues/3709

[skip changelog]

### CODE REVIEW

**Review by dracos — CHANGES_REQUESTED** [review]
It looks like open311-adapter Confirm and Alloy are the main ones that do something with a photo on an update. I assume we're not changing Confirm as part of this (not sure what it would do), so we only want this change for Alloy? So I guess I'm saying you need a different flag to check to do this direct attachment of photo in an update, where you know the other end can handle it. Your new `_add_photos_to_upload` isn't checking `upload_files` as `_populate_service_request_uploads` does, so it should probably be doing that, but also have a new flag upload_files_updates or something better, to check, and then can be added to Open311Alloy?

**Comment by MorayMySoc** [comment]
I'm thinking that now, testing for `upload_files_for_updates` is tested before getting to` _add_photos_to_upload` so should be ok

File: t/sample.jpg
Comment (dracos) [inline t/sample.jpg:1]: Not sure why this has been added? :)

**Review by dracos — APPROVED** [review]
Looks good, not sure why the extra image is present though is all.

**Comment by MorayMySoc** [comment]
Not merged as broke master so had to reset master

**Pull request merged**